uast/uast.go

```diff
@@ -33,6 +33,21 @@ const (
 	// File is the root node of a single file AST.
 	File
 
+	// Binary aritmethic operators. Examples with C operators.
+	// TODO: should we have division and FloorDivision like Python or Nim?
+	// TODO: should we had the pow operator that some languages have?
+
+	// OpArithmeticAdd is the binary add operator (i.e. + in most languages)
```

Comment:
End all comments with a full stop, they are ordinary English phrases that will be used verbatim in the documentation so they must obey standard English language rules.